src/pages/Poster.tsx

```diff
@@ -64,12 +60,26 @@ const Poster = () => {
   useEffect(() => {
     const imdbId = data?.movie.imdbId
     const title = data?.movie.title
-    const posterUrl = data?.movie.posterPath
-    if (imdbId && title && posterUrl) {
+    const posterPath = data?.movie.posterPath
+    const budget = data?.movie.budget
+    const popularity = data?.movie.popularity
+    const revenue = data?.movie.revenue
+    console.log({
+      imdbId,
+      title,
+      posterPath,
+      budget,
+      popularity,
+      revenue,
+    })
+    if (imdbId && title && posterPath && budget && popularity && revenue) {
```

Comment:
I would assume that all of these are set if data.movie is set. Otherwise, if they weren't, we wouldn't get any movie anyways and the app would break (as we do not retry the request or anything).

Or just don't require budget etc. and handle these missing failures appropriately (e.g., don't show the stat on the other pages).